apps/core/src/migration/migration.service.ts

```diff
@@ -250,8 +291,11 @@ export class MigrationService {
   }
 
   public getMigrationStatus(): MigrationStatus {
-    // When migration is complete, set it back to NOT_STARTED state
-    if (this.migrationStatus.status === Status.COMPLETE) {
+    // When migration is complete (success or error), set it back to NOT_STARTED state
+    if (
+      this.migrationStatus.status === Status.COMPLETE ||
+      this.migrationStatus.status === Status.ERROR
```

Comment:
nit: the wording of `complete` collides with `Status.COMPLETE` (success) here; `settled` would be the inclusive word (as seen in `Promise`).

Reply:
yeah that makes snese, updated in new commit push